plugins/node/opentelemetry-instrumentation-aws-sdk/.tav.yml

```diff
@@ -1,21 +1,21 @@
 "aws-sdk":
   # there are so many version to test, it can take forever.
   # we will just sample few of them
-  versions: ">=2.1230.0 || 2.1219.0 || 2.1152.0 || 2.1048.0 || 2.1012.0 || 2.647.0 || 2.308.0"
+  versions: ">=2.1266.0 || 2.1262.0 || 2.1219.0 || 2.1048.0 || 2.1012.0 || 2.647.0 || 2.308.0"
   commands:
     - npm run test
   # Fix missing `contrib-test-utils` package
   pretest: npm run --prefix ../../../ lerna:link
 
 "@aws-sdk/client-s3":
-  versions: ">=3.188.0 || 3.154.0 || 3.107.0 || 3.54.0 || 3.6.1"
+  versions: ">=3.223.0 || 3.218.0 || 3.216.0 || 3.154.0 || 3.107.0 || 3.54.0 || 3.6.1"
   commands:
     - npm run test
   # Fix missing `contrib-test-utils` package
   pretest: npm run --prefix ../../../ lerna:link
 
 "@aws-sdk/client-sqs":
-  versions: ">=3.188.0 || 3.171.0 || 3.107.0 || 3.58.0 || 3.54.0 || 3.43.0 || 3.24.0"
+  versions: ">=3.216.0 || 3.171.0 || 3.58.0 || 3.54.0 || 3.43.0 || 3.24.0"
   commands:
```

Comment:
Might be helpful to document in a comment that version `3.192.0` is broken and link to [this issue](https://github.com/aws/aws-sdk-js-v3/issues/4060)